### ISSUE TITLE
error: event bus is closed

### ISSUE THREAD

**Issue by b5** [qri-io/qri#1474]
Every now & then this message creeps up into userland:

```
2020-07-29T22:57:34.556-0400	ERROR	lib	lib/lib.go:522	event bus is closed
```

This is printed whenever an event is published on the bus _after_ the bus context is closed. Here's the origin point:

https://github.com/qri-io/qri/blob/921da6399a2a1a659a33a9755e5884a4fae34c66/event/event.go#L113

This one is tough to avoid. In production the event bus is wired to a context that should span the lifetime of the process, and _any_ subsystem that publishes to the bus needs to stop doing publishing stuff after that context is cancelled, which means properly wiring up the right context in all subsystems, and properly handling all goroutines.

### Steps to fix
* do an audit & clean of all process-spanning contexts until the message disappears from all tests
* move this message to debug level
* properly wire up & honor the process-spanning context across all subsystems that publish to the bus

That last one is ongoing, so I think this issue should be closed once we've done the first two

**Comment by b5** [comment]
cc @feep

**Comment by dustmop** [comment]
Should this be changed to being a log.Info instead? It's mainly intended for developers, who can run `--log-all` or whatever, and there's no actionable response that other users can do about it.

**Comment by b5** [comment]
yeah, I'd use  `log.Debug` instead, which is what our `--log-all` sets logging levels to
